### DISCUSSION TITLE
dubbo 2.7.8版本使用kryo序列化比默认的hessian2性能更差

### DISCUSSION THREAD

**Discussion by axin3892** [apache/dubbo#13903] [Question]
### Pre-check

- [ ] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

按照这个文档https://cn.dubbo.apache.org/zh-cn/docsv2.7/user/serialization/      来使用kryo序列化，结果发现比默认的序列化性能更差，具体表现为接口耗时增加了15ms左右，cpu使用率也高于默认序列化
![image](https://github.com/apache/dubbo/assets/52096123/1db105e3-2264-4656-aecf-d38daf605304)    cpu使用率分别为8%和11%左右
![image](https://github.com/apache/dubbo/assets/52096123/db2e3fa7-eee8-4851-8c37-f64c14163817)    
![image](https://github.com/apache/dubbo/assets/52096123/7a3f7964-91a2-43c9-89c0-102779775a77)
请问这是为什么，kryo默认配置理论应该也比hessian好一点才对吧，为啥反而是更差，而且差的还挺明显

### Code of Conduct

- [ ] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Please re-submit the discussion in English. GitHub Discussion is only for English.
